### ISSUE TITLE
Previous/next page arrow buttons are hidden, but associated commands are enabled (invokable by keyboard shortcuts and access keys) => problematic with non-linear spine items

### ISSUE THREAD

**Issue by danielweck** [readium/readium-js-viewer#188]
To reproduce:
(1) navigate to a non-linear spine item (via the TOC or an content-embedded hyperlink) => the left/right arrow buttons are correctly hidden.
(2) use the arrow keys to switch page => the non-linear XHTML document should be "trapped" as if it were represented in a modal dialog, but the page switches unexpectedly.

Suggested solutions:

(1) disable the actual page turn command, not just visually hide the buttons. Consequently though, the user would need to be able to navigate back to the previous document CFI location (i.e. which was displayed when the hyperlink to the non-linear document was activated). iBooks implements a proper modal dialog, with a "close" cross icon to discard the fullscreen popup overlay. In Readium however, another type of UI affordance needs to be introduced in order to achieve a similar effect. I'm not sure how, but accessibility is paramount, as non-linear documents are often used for ancillary content descriptions. Thoughts? (this would need to be implemented in v1, otherwise non-linear spine items would be like "trapping" blackholes!)

(2) enable both page arrow buttons when navigating to a non-linear document. This way, we revert back to the original behaviour and we decide to tackle the problem in v1+, see:
https://github.com/readium/readium-shared-js/issues/26

I have a preference for (1), but I am not sure whether this development task is achievable within the v1 timeframe. @rkwright ?

**Comment by danielweck** [comment]
Fixed in PR: https://github.com/readium/readium-js-viewer/pull/402

**Comment by muyarkyawwin** [comment]
May I know how to disable the Previous and Next buttons in Readium?

**Comment by danielweck** [comment]
@muyarkyawwin `$("#ID").show()` / `$("#ID").hide()` is the naive way of doing it, as described in this issue (the correct way is to implement command binding, so that the state of the UI is refreshed based on the internal state of the navigation commands (e.g. disabled / enabled)).

See:
https://github.com/readium/readium-js-viewer/blob/develop/src/js/EpubReader.js#L752
```javascript
    var updateUI = function(pageChangeData){
        if(pageChangeData.paginationInfo.canGoLeft())
            $("#left-page-btn").show();
        else
            $("#left-page-btn").hide();
        if(pageChangeData.paginationInfo.canGoRight())
            $("#right-page-btn").show();
        else
            $("#right-page-btn").hide();
    }
```
